<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FLCreateJobDlg</class>
 <widget class="QDialog" name="FLCreateJobDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create new job</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Job detail</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="1">
       <widget class="QTextEdit" name="leJobComments">
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;New Job&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cbJobStandard">
        <item>
         <property name="text">
          <string>UL(United States)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ULC(Canada)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Job Standard</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Set as default path</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Comment</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Job Path</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leJobName">
        <property name="text">
         <string>NewJob</string>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leJobPath">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pbJobPath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Job Name</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>leJobName</zorder>
     <zorder>cbJobStandard</zorder>
     <zorder>label_2</zorder>
     <zorder>pbJobPath</zorder>
     <zorder>label_4</zorder>
     <zorder>label</zorder>
     <zorder>leJobPath</zorder>
     <zorder>leJobComments</zorder>
     <zorder>label_5</zorder>
     <zorder>checkBox</zorder>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Job Floors</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="4">
       <widget class="FLTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pbAddFloor">
        <property name="text">
         <string>Add Floor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pbDeleteFloor">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete Floor</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pbEditFloor">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Edit Floor</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>52</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>288</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FLTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>fltreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="firelink.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>FLCreateJobDlg</receiver>
   <slot>on_item_clicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leJobName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FLCreateJobDlg</receiver>
   <slot>on_jobname_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_item_clicked(QTreeWidgetItem*,int)</slot>
  <slot>on_jobname_changed(QString)</slot>
 </slots>
</ui>
